.cz-config.js

```diff
@@ -0,0 +1,49 @@
+const { execSync } = require('child_process');
+
+const getPackages = () => {
+  let outputPackages;
+
+  try {
+    outputPackages = execSync('npx turbo run build --dry=json');
+  } catch (error) {
+    console.info('No changed packages found.');
+    process.exit(0);
+  }
+
+  const packagesArray = JSON.parse(outputPackages.toString());
+
+  return packagesArray.packages.map((packageName) => packageName.replace('@justeattakeaway/', ''));;
+};
+
+module.exports = {
+  // We have to provide '\u0020' to some emojis due to a spacing bug with commitizen - https://github.com/commitizen/cz-cli/issues/815
+  types: [
+    { value: 'feat', name: '✨ feat: A new feature or functionality' },
+    { value: 'fix', name: '🐛 fix: A bug fix' },
+    { value: 'chore', name: '🧹 chore: Changes that affect the build system or external dependencies (e.g: typescript, webpack, eslint)' },
+    { value: 'ci', name: '⚙️ \u0020ci: Changes to our CI configuration files and scripts' },
```

Comment:
Should this have a unicode string here? Just wondering as non of the other values do.

Reply:
Yep, there's a comment provided above 👍🏼 

We have to provide '\u0020' to some emojis due to a spacing bug with commitizen - https://github.com/commitizen/cz-cli/issues/815